core/src/apps/solana/types.py

```diff
@@ -4,8 +4,11 @@
     from enum import IntEnum
     from typing import Any, Callable, Generic, TypeVar
 
+    from src.trezor.messages import SolanaTxAdditionalInfo, SolanaTxTokenAccountInfo
     from trezor.utils import BufferReader
+    from typing_extensions import Self
 
+    from .definitions import Definitions
```

Comment:
This one needs to be moved out of `if TYPE_CHECKING:` since we're using it during runtime.